apps/settings/js/modules/storage/volume.js

```diff
@@ -0,0 +1,155 @@
+/**
+ * Volume:
```

Comment:
I'm not sure why we use `volume` in the very beginning. It seems make more sense to use `DeviceStorage`.